test/utilities/WindowTimer.js

```diff
@@ -11,7 +11,13 @@ export default class {
     this._clears = 0
   }
 
-  /** @return {?function} The timeout function set by setTimeout(). */
+  /**
+   * Timeout function.
+   *
+   * @callback timeoutFunction
```

Comment:
Nice! What do you think about using a capital T since this is kind of like a type? The docs use the same style you've shown here but also mention `@callback` is very similar to `@typedef`. Same suggestion anywhere else `@callback` is used too.

http://usejsdoc.org/tags-callback.html

Reply:
Fixed!